Automattic/sensei
# Remove compat gutenberg

### Changes proposed in this Pull Request

* Deprecate compact scripts that we used to < 5.4.

### Testing instructions

* Just test import, export, and setup wizard, and make sure it continues working properly.


### Deprecated Code

* `Sensei_Assets::override_script`
* `Sensei_Assets::wp_compat`

~~When we remove these methods, we should remove the `assets/vendor/gutenberg` folder too.~~
EDIT: See the following discussions, but since we removed the `wp_compat` method content, there is no method importing that assets anymore.
